react/react.d.ts

```diff
@@ -124,7 +124,7 @@ declare namespace __React {
         render(): JSX.Element;
         props: P;
         state: S;
-        context: {};
+        context: { [key: string]: any };
```

Comment:
Don't do this. You should declare the type of `context` on your component class.